src/package/vsix/CMakeLists.txt

```diff
@@ -1,18 +1,20 @@
 cmake_minimum_required(VERSION 3.9)
 
 # The C++/WinRT VSIX package is only targeted at Visual Studio (on Windows)
-if (WIN32 AND vssdk_installed)
+if (WIN32 AND ("${XLANG_BUILD_ARCH}" STREQUAL "x86"))
+
+get_target_property(cppwinrt_natvis_dir_x86 cppwinrtvisualizer "cppwinrt_natvis_dir")
+string(REGEX REPLACE "x86" "x64" cppwinrt_natvis_dir_x64 ${cppwinrt_natvis_dir_x86})
 
-get_target_property(cppwinrt_natvis_dir cppwinrtvisualizer "cppwinrt_natvis_dir")
 get_target_property(cppwinrt_nupkg_dir make_cppwinrt_nupkg "cppwinrt_nupkg_dir")
 
 file(TO_NATIVE_PATH ${CMAKE_CURRENT_BINARY_DIR}/Microsoft.Windows.CppWinRT.vsix cppwinrt_vsix)
 file(TO_NATIVE_PATH "${cppwinrt_nupkg_dir}/Microsoft.Windows.CppWinRT.${XLANG_BUILD_VERSION}.nupkg" cppwinrt_nupkg)
 
-set(build_vsix msbuild ${CMAKE_CURRENT_SOURCE_DIR}/vsix.csproj /nologo /m /p:Configuration=${CMAKE_BUILD_TYPE},Platform=$ENV{VSCMD_ARG_TGT_ARCH})
+set(build_vsix msbuild ${CMAKE_CURRENT_SOURCE_DIR}/vsix.csproj /nologo /m /p:Configuration=${CMAKE_BUILD_TYPE},Platform=${XLANG_BUILD_ARCH})
```

Comment:
Are we using msbuild from cmake here for convienence (i.e. we don't need to compile this xplat) or is there something specific for vsix scenarios that cmake doesn't support?

Reply:
We're using it for both vsix and now natvis, due to lots of magic behavior in VS project file support for both.  the alternative, reinventing all this in cmake for windows-specific builds, makes little sense and introduces friction.

Reply:
(Natvis builds now require a VS SDK NuGet, as of VS 16)

Reply:
👍